src/rp2/plugin/accounting_method/fifo.py

```diff
@@ -63,3 +64,12 @@ def seek_non_exhausted_acquired_lot(
 
     def lot_candidates_order(self) -> AcquiredLotCandidatesOrder:
         return AcquiredLotCandidatesOrder.OLDER_TO_NEWER
+
+    def heap_key(self, lot: InTransaction) -> float:
+        raise ValueError()
+
+    def use_heap(self) -> bool:
+        return False
```

Comment:
It would be great not to have heap-related methods in FIFO (see my other comment at line 57 of abstract_accounting_method.py on this topic).